frontend/www/js/omegaup/csv.d.ts

```diff
@@ -11,5 +11,16 @@ declare module '@/third_party/js/csv.js/csv.js' {
       commentChar: string;
     };
   }
+  interface Source {
+    file: File | null;
+    encoding?: string;
+    url?: string;
+    data?: any;
+  }
+  export interface Dataset {
+    fields: Array<string>;
+    records: Array<any>;
```

Comment:
```suggestion
    fields?: string[];
    records: (null | string | number)[][];
```

hay una rama de [`my.extractFields()`](https://github.com/okfn/csv.js/blob/b003d775cc010e425cfe8c2e5c71af637c4d3ebb/csv.js#L101-L112) donde no se establece `fields`, así que hay que marcarla como opcional. y por último, `records` sólo puede ser un arreglo de arreglos de tipos que regresa [`processField`](https://github.com/okfn/csv.js/blob/b003d775cc010e425cfe8c2e5c71af637c4d3ebb/csv.js#L160-L178): `null`, cadenas y números.

Reply:
Listo